### ISSUE TITLE
(windows 11) - Keeps crashing when running simple search query

### ISSUE THREAD

**Issue by Emiltayeb** [calclavia/mcp-obsidian#3]
![image](https://github.com/user-attachments/assets/81f01cd8-5d4a-4f00-b147-ecc654b22285)

it happends everytime, no matter what im asking. 
It thinkgs for a while, saying it found some matches but then crashes.

Dosent happend in other MPC.

**Comment by calclavia** [comment]
This seems like it's from Claude's side, not related to this MCP.

**Comment by Emiltayeb** [comment]
> This seems like it's from Claude's side, not related to this MCP.

Dosent happend in any other MPC.
everything u else it's fine.

**Comment by danyx23** [comment]
I see the same issue, also on Windows 11. The `search_notes` invocation works and gives a result but then read_notes is apparently invoked with an empty `paths`:
```
{
  `paths`: []
}
No result received from client-side tool execution.
```

I don't see anything obviously wrong with the code at first glance. I guess you are developing this on MacOS and it works there?

**Comment by calclavia** [comment]
@danyx23 Yes, I'm developing this on a Mac. If you're on Windows and have a fix, would appreciate a PR!

**Comment by BindingOx** [comment]
any fix for crashing on windows?

**Comment by calclavia** [comment]
https://github.com/smithery-ai/mcp-obsidian/issues/13
